internal/outgoing/dispatcher.go

```diff
@@ -321,16 +339,20 @@ func (d *Dispatcher) processDelayedEvent(ctx context.Context, e models.IncomingE
 
 		// Schedule send event later
 		dispatcher.TrackAfterFuncJob(timeRemain, func() {
```

Comment:
maybe should make a new span for this AfterFunc, so we can simple `defer span.End()` for the original span